app/controllers/home_controller.rb

```diff
@@ -51,6 +51,7 @@ def dashboard
                           .count
     @user_note_count = Node.where(type: 'note', status: 1, uid: current_user.uid).count if current_user
     self.activity
+    @trending_tags = Tag.trending 
```

Comment:
Here, I feel like we might more easily just put `Tag.trending` in the view code directly. Note that the template is used in a few places, so this may account for some of your errors, and just having the template directly call the method may be better.